### PR TITLE
Algolia - Handle settings in code

### PR DESCRIPTION
**Pull request by JorindeUsMedia**
- Add node script to set algolia settings
- Set index settings which are not algolia-default

### CODE REVIEW

File: hugo/algolia-settings.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+if (!indexName) {
+    throw new Error('Please provide `process.env.ALGOLIA_INDEX_NAME`');
+}
+
+const adminKey = process.env.ALGOLIA_ADMIN_KEY;
+if (!adminKey) {
+    throw new Error('Please provide `process.env.ALGOLIA_ADMIN_KEY`');
+}
+
+const client = algoliasearch('5LXFM0O81Q', adminKey);
```
Comment (myitcv) [inline hugo/algolia-settings.js:13]: Available as `ALGOLIA_APP_ID`

**Review by myitcv — APPROVED** [review]
LGTM with one suggestion.

File: hugo/algolia-settings.js (hunk 1 above)
Comment (JorindeUsMedia) [reply hugo/algolia-settings.js]: Changed it to use the env var.

**Review by myitcv — APPROVED** [review]
LGTM. Will import.

**Comment by myitcv** [comment]
Closed via 5db31aa3e826bdf0d97e334839b51dbccf80e15b

**Pull request closed without merging**